Agents/MetOfficeAgent/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.8"
+
+services:
+  metoffice_agent_debug:
```

Comment:
There are standard ways to removed the need for duplication in docker-compose files but I'm not sure it's worth changing things at this point.